specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json

```diff
@@ -1122,7 +1122,13 @@
         },
         "enableSoftDelete": {
           "type": "boolean",
-          "description": "Property to specify whether the 'soft delete' functionality is enabled for this key vault. If omitted, assume true as default value. Once set to true, cannot be reverted to false."
+          "default": true,
+          "description": "Property to specify whether the 'soft delete' functionality is enabled for this key vault. If it's not set to any value(true or false) when creating new key vault, it will be set to true by default. Once it's been set to true value, it can NOT be reverted to false."
+        },
+        "softDeleteRetentionInDays": {
```

Comment:
>softDeleteRetentionInDays [](start = 9, length = 25)

What's the default value on `softDeleteRetentionInDays` and can we put it in the swagger?

Reply:
90

Reply:
Thanks for updating